test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -154,6 +163,10 @@ export default class BaseFormObject {
   }
 
   getSaveButtonLabel() {
+    if (userVariables.v3) {
+      return within(this.el).getByRole('button').textContent;
+    }
+
     return within(this.el).getByRole('button').value;
```

Comment:
Does it break other tests to use `textContent` instead of `value`? Wondering if we could collapse these down.

Reply:
yeah in v3 the buttons dont use a value prop where in v2 buttons only have a value and no textContent.